src/NewTools-Debugger/StDebugger.class.st

```diff
@@ -144,12 +145,37 @@ StDebugger class >> debuggerContextClass [
 	^ StDebuggerContext
 ]
 
+{ #category : #accessing }
```

Comment:
In the debugger the settings should be under the `settings` protocol

Reply:
Good catch thanks :)